CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -828,6 +859,24 @@ void IRenderWebBrowser.OnImeCompositionRangeChanged(Range selectedRange, Rect[]
         protected virtual void OnImeCompositionRangeChanged(Range selectedRange, Rect[] characterBounds)
         {
             //TODO: Implement this
+            if (_imeWin != null)
+            {
+                UiThreadRunAsync(() =>
+                {
+                    IList<Rect> rects = new List<Rect>();
+                    foreach (var item in characterBounds)
+                    {
+                        GeneralTransform generalTransform1 = TransformToAncestor(this);
+
+                        if (CleanupElement != null)
+                        {
+                            Point point = this.TransformToAncestor(CleanupElement).Transform(new Point(0, 0));
```

Comment:
Using `CleanupElement` here really isn't a reliably method to access the parent `Window`, which is what I'm assuming your doing. I'm sure you can find a better way of doing this.